scripts/build.ts

```diff
@@ -39,3 +45,25 @@ function buildPackage() {
     encoding: "utf-8",
   });
 }
+
+function buildExtendedJSON() {
+  // TODO: Validate the resulting JSON against a schema.
+  for (const [id, featureData] of Object.entries(data.features)) {
+    if (Array.isArray(featureData.compat_features) && featureData.status) {
+      const by_compat_key = {};
+
+      for (const key of featureData.compat_features) {
+        by_compat_key[key] = { status: getStatus(id, key) };
+      }
+
+      if (Object.keys(by_compat_key).length) {
```

Comment:
Is this check really needed? How about just asserting `featureData.compat_features.length` earlier on?

(I don't love `Object.keys(...).length` so am looking for a way to avoid it.)

Reply:
Fixed with c95aefd74bf5ea6226215b85bdff985dd6aab09f